static/src/stylesheets/layout/new-header/_pillars.scss

```diff
@@ -35,8 +35,13 @@
     display: inline-block;
 
     @include mq(desktop) {
+        min-width: 0;
+        padding-right: 16px;
+        transition: min-width 600ms ease;
```

Comment:
I kind of think this should be quicker, it feels a bit slow to me. What do you think?